Tests/DependencyInjection/LuneticsLocaleExtensionTest.php

```diff
@@ -50,14 +52,6 @@ public function testLoad($configs, $strictMatch)
         }
 
         $this->assertEquals($strictMatch, $container->hasDefinition('lunetics_locale.best_locale_matcher'));
-
-        $resources = $container->getResources();
-
-        $this->assertContains('validator.xml', $resources[0]->getResource());
```

Comment:
i don't think this test adds value. we test that the services exist, that is enough imho. if we want more, we should instead have functional tests with a small test symfony application to make sure things actually work.